SimakovaKY/lab work 1.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream> 
+#include <conio.h>
+using namespace std;
+char ch = _getch();
```

Comment:
There is no point in asking user for input here, as it should be done each time one cycle of your program is complete